### PR TITLE
Add dependency injection

### PR DESCRIPTION
**Pull request by Jaunius343**

### CODE REVIEW

File: PSI Food waste/Startup.cs [hunk 1]
```diff
@@ -23,13 +23,24 @@ public Startup(IConfiguration configuration)
...
+        //}
+
         public IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddRazorPages();
             services.AddSession();
+            services.AddSingleton<IEmployeeRepository, MockEmployeeRepository>();
```
Comment (Pachtakoras) [inline PSI Food waste/Startup.cs:41]: These interfaces dont exist so program doesn't compile

File: PSI Food waste/Pages/Forms/RestaurantVerified.cshtml.cs [hunk 2]
```diff
@@ -55,7 +43,7 @@ public void OnGet()
         {
             ViewData["User"] = HttpContext.Session.GetString(key: "username");
             _productRepository.SortProducts();
-            products = _productRepository.GetList(_restaurantRepository.GetNextID());
+            products = _productRepository.GetList(_restaurantRepository.GetID());
```
Comment (Pachtakoras) [inline PSI Food waste/Pages/Forms/RestaurantVerified.cshtml.cs:47]: Yes, i was about to mention that GetNextId doesnt exist.

File: PSI Food waste/Startup.cs (hunk 1 above)
Comment (Jaunius343) [reply PSI Food waste/Startup.cs]: Yes, I totally forgot to delete them after testing

File: PSI Food waste/Pages/Forms/RestaurantVerified.cshtml.cs (hunk 2 above)
Comment (Jaunius343) [reply PSI Food waste/Pages/Forms/RestaurantVerified.cshtml.cs:47]: mhm, I changed the name to make more sense

**Pull request merged**